### PR TITLE
Contact blocked stopped fields

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: flows/contact.go [hunk 1]
```diff
@@ -130,6 +140,18 @@ func (c *Contact) SetLanguage(lang envs.Language) { c.language = lang }
 // Language gets the language for this contact
 func (c *Contact) Language() envs.Language { return c.language }
 
+// IsBlocked returns whether the contact is blocked or not
+func (c *Contact) IsBlocked() bool { return c.isBlocked }
```
Comment (rowanseymour) [inline flows/contact.go:144]: `Blocked` / `SetBlocked` / `Stopped` / `SetStopped` (go convention does prefix getters with `get` or `is` etc)

File: flows/events/contact_is_blocked_changed.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package events
+
+import (
+	"github.com/nyaruka/goflow/flows"
+)
+
+func init() {
+	registerType(TypeContactIsBlockedChanged, func() flows.Event { return &ContactIsBlockedChangedEvent{} })
```
[the remark this answers is not in the document]
Comment (norkans7) [reply flows/events/contact_is_blocked_changed.go:8]: so these should us the `Contact` affected and dump it in the event?

[the remark this answers is not in the document]
Comment (norkans7) [reply flows/events/contact_is_blocked_changed.go:8]: I assume we add the `state` only in that event

[the remark this answers is not in the document]
Comment (rowanseymour) [reply flows/events/contact_is_blocked_changed.go:8]: No I think these can be empty events.. the type of the event tells us everything we need to know

**Comment by norkans7** [comment]
Replaced by https://github.com/nyaruka/goflow/pull/902

**Pull request closed without merging**